internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -159,18 +158,14 @@ NLBCertManagerRole:
             - sts:AssumeRole
     Path: /
     Policies:
-      - PolicyName: "NLBCertManagerPolicyWrite"
+      - PolicyName: "NLBCertManagerPolicy"
         PolicyDocument:
           Version: '2012-10-17'
           Statement:
             - Sid: EnvHostedZoneUpdateAndWait
               Effect: Allow
               Action: route53:ChangeResourceRecordSets
-              Resource:
```

Comment:
porque?

Reply:
This was a mistake during rebasing. I thought this was the  "route53:GetChange" policy statement, which needs `Resource: "*"`. Fixed!